CMakeLists.txt

```diff
@@ -69,10 +70,16 @@ endif()
 
 
 # Use ccache when available to speed up builds.
-find_program(CCACHE_FOUND ccache)
-if(CCACHE_FOUND)
-  set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE ccache)
-  message(STATUS "using ccache")
+if (USE_CCACHE)
+  find_program(CCACHE_FOUND ccache)
+  if(CCACHE_FOUND)
+    set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE ccache)
+    message(STATUS "using ccache")
+  else()
+    message(STATUS "ccache not found")
+  endif()
+else()
+  message(STATUS "ccache not enabled")
```

Comment:
This last message has very little information:
Imagine you try to build and see it. Would you understand what it means?
I guess it's not too important as you actively need to change a default to get there, but still it might be worth improving.
How about "ccache disabled; set option USE_CCACHE to use ccache if available"?

Reply:
I'm ok with new text, but it doesn't make sense, because `USE_CCACHE` is on by default => you will see this message only if explicitly specified `-D USE_CCACHE=OFF`

Reply:
Text has been clarified.